src/modules/attitude_estimator_q/attitude_estimator_q_main.cpp

```diff
@@ -400,6 +400,35 @@ void AttitudeEstimatorQ::task_main()
 
 			if (!_failsafe) {
 				uint32_t flags = DataValidator::ERROR_FLAG_NO_ERROR;
+#ifdef __PX4_POSIX
```

Comment:
This section is getting duplicated and a merge fail. I don't think this was the original intent.